src/components/Filters/FiltersHydrator.js

```diff
@@ -14,7 +14,7 @@ const FiltersHydrator = ({ namespace, onHydrated }) => {
     if (isHydrated) {
       dispatch(actions.filtersHydrated(namespace));
 
-      if (onHydrated) {
+      if (!hydrated && onHydrated) {
```

Comment:
What's `isHydrated`? Doesn't it mean that `hydrated` is true?

Reply:
No, one is local state, other is redux state. They do have gaps, but If I already sent an action I don't need to hydrate the second time.

I played with the demo to check things are okay.

Reply:
Rolled back this change. Seems I had a mistake here.